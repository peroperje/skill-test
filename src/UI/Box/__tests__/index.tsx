import React from 'react';
import { create } from 'react-test-renderer';

import { Box } from '../index';

describe('Box', () => {
  it('Snapshot', () => {
    const component = create(<Box />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
