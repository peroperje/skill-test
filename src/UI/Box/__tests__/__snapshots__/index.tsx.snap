// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box Snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
/>
`;
